ale/drivers/mdis_driver.py

```diff
@@ -127,10 +128,35 @@ def _exposure_duration(self):
     @property
     def target_name(self):
         return self._label['IsisCube']['Instrument']['TargetName']
-        
+
     @property
     def starting_ephemeris_time(self):
         if not hasattr(self, '_starting_ephemeris_time'):
             sclock = self._label['IsisCube']['Archive']['SpacecraftClockStartCount']
             self._starting_ephemeris_time = spice.scs2e(self.spacecraft_id, sclock)
         return self._starting_ephemeris_time
+
+class isisSpicedMessenger(Cube, TransverseDistortion):
```

Comment:
Pedantic - class names should be capitalized from the start.

Kudos to all that have worked on this driver structure - this is very clean looking.